examples/cassandra/README.md

```diff
@@ -176,78 +138,38 @@ spec:
-The important thing to note here is the `selector`. It is a query over
-labels, that identifies the set of _Pods_ contained by the _Service_.  In this
-case the selector is `app=cassandra`.  If you look back at the Pod
-specification above, you'll see that the pod has the corresponding label, so it
-will be selected for membership in this Service.
+An important thing to note here is the `selector`. It is a query over labels,
+that identifies the set of Pods contained by this Service.  In this case the
+selector is `app=cassandra`. If there are any pods with that label, they will be
+selected for membership in this service. We'll see that in action shortly.
 
-Create this service as follows:
+Create the Cassandra service as follows:
 
 ```console
 $ kubectl create -f examples/cassandra/cassandra-service.yaml
 ```
 
-Now, as the service is running, we can create the first Cassandra pod using the mentioned specification.
 
-```console
-$ kubectl create -f examples/cassandra/cassandra.yaml
-```
-
-After a few moments, you should be able to see the pod running, plus its single container:
-
-```console
-$ kubectl get pods cassandra
-NAME        READY     STATUS    RESTARTS   AGE
-cassandra   1/1       Running   0          55s
-```
-
-You can also query the service endpoints to check if the pod has been correctly selected.
+## Step 2: Use a Replication Controller to create Cassandra node pods
 
-```console
-$ kubectl get endpoints cassandra -o yaml
-apiVersion: v1
-kind: Endpoints
-metadata:
-  creationTimestamp: 2015-06-21T22:34:12Z
-  labels:
-    app: cassandra
-  name: cassandra
-  namespace: default
-  resourceVersion: "944373"
-  selfLink: /api/v1/namespaces/default/endpoints/cassandra
-  uid: a3d6c25f-1865-11e5-a34e-42010af01bcc
-subsets:
-- addresses:
-  - ip: 10.244.3.15
-    targetRef:
-      kind: Pod
-      name: cassandra
-      namespace: default
-      resourceVersion: "944372"
-      uid: 9ef9895d-1865-11e5-a34e-42010af01bcc
-  ports:
-  - port: 9042
-    protocol: TCP
-```
-
-### Adding replicated nodes
-
-Of course, a single node cluster isn't particularly interesting.  The real power
-of Kubernetes and Cassandra lies in easily building a replicated, scalable
-Cassandra cluster.
+As we noted above, in Kubernetes, the atomic unit of an application is a
+[_Pod_](../../docs/user-guide/pods.md).
+A Pod is one or more containers that _must_ be scheduled onto
+the same host.  All containers in a pod share a network namespace, and may
+optionally share mounted volumes.
 
-In Kubernetes a
+A Kubernetes
 _[Replication Controller](../../docs/user-guide/replication-controller.md)_
 is responsible for replicating sets of identical pods.  Like a
-_Service_, it has a selector query which identifies the members of its set.
-Unlike a _Service_, it also has a desired number of replicas, and it will create
-or delete _Pods_ to ensure that the number of _Pods_ matches up with its
+Service, it has a selector query which identifies the members of its set.
```

Comment:
What are the members of its set?  How does this differ from the pods it is replicating?  Does this mean it identifies the pods it has created from the template using a selector?